pvlib/ivtools.py

```diff
@@ -0,0 +1,262 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Mar 29 10:34:10 2019
+
+@author: cwhanse
+"""
+
+import numpy as np
+
+
+def fit_cec_with_sam(celltype, v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
+                     gamma_pmp, cells_in_series, temp_ref=25):
+    '''
+    Estimates parameters for the CEC single diode model using the SAM SDK.
+
+    Parameters
+    ----------
+    celltype : str
+        Value is one of 'monoSi', 'multiSi', 'polySi', 'cis', 'cigs', 'cdte',
+        'amorphous'
+    v_mp : float
+        Voltage at maximum power point at standard test condition (STC)
```

Comment:
Here and elsewhere. I think STC should more generally be called reference conditions (RC), which is used in the outputs. If I understand correctly, nothing about this fit requires that RC be STC.

Reply:
I think that change is a little risky. Unlike the reference temperature, SAM provides no keyword to specify the reference irradiance condition, and the [documentation](https://www.nrel.gov/docs/fy15osti/64102.pdf) page 43 shows 1000 W/m2 explicitly in the equations for `R_SH` and `I_L`. For this reason, I think the input values `v_mp` etc. have to be at 1000 W/m2 and the reference temperature

Reply:
If the reference temperature is not the default 25 degC, then STC is a misnomer, correct?

Reply:
Yes, it is. I looked through the NREL source to confirm that 1) Tref is used and 2) no irradiance is assumed in the fitting.  Changing the docstring and adding a note about how the reference condition is implicitly specified.